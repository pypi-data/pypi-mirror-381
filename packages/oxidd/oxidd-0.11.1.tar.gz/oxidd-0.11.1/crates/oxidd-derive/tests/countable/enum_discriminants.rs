use oxidd_derive::Countable;

#[derive(Countable)]
#[repr(u8)]
enum EdgeTag {
    None = 0,
    Complemented = 1,
}

fn main() {}
