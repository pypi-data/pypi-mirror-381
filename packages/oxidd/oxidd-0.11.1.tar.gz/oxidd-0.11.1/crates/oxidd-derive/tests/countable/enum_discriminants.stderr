error: cannot derive `Countable` for `enum`s with explicit discriminants
 --> tests/countable/enum_discriminants.rs:6:12
  |
6 |     None = 0,
  |            ^

error: cannot derive `Countable` for `enum`s with explicit discriminants
 --> tests/countable/enum_discriminants.rs:7:20
  |
7 |     Complemented = 1,
  |                    ^
